swap/src/main.rs

```diff
@@ -14,25 +14,31 @@
 #![allow(non_snake_case)]
 
 use anyhow::{bail, Context, Result};
+use cli::{Command, Options, Resume};
+use config::Config;
+use database::{Database, Swap};
 use libp2p::{core::Multiaddr, PeerId};
+use network::transport::build;
 use prettytable::{row, Table};
+use protocol::{alice, alice::AliceState, bob, bob::BobState, SwapAmounts};
 use rand::rngs::OsRng;
 use std::sync::Arc;
 use structopt::StructOpt;
-use swap::{
-    bitcoin,
-    cli::{Command, Options, Resume},
-    config::Config,
-    database::{Database, Swap},
-    monero,
-    network::transport::build,
-    protocol::{alice, alice::AliceState, bob, bob::BobState},
-    trace::init_tracing,
-    SwapAmounts,
-};
+use trace::init_tracing;
 use tracing::{info, log::LevelFilter};
 use uuid::Uuid;
 
+pub mod bitcoin;
+pub mod cli;
+pub mod config;
+pub mod database;
+pub mod monero;
+pub mod network;
+pub mod protocol;
+#[cfg(test)]
+mod tests;
+pub mod trace;
+
 #[macro_use]
 extern crate prettytable;
```

Comment:
Is this needed? I don't think the `#[macro_use]` and `extern` keywords are necessary anymore.